config/initializers/zeitwerk.rb

```diff
@@ -0,0 +1,3 @@
+require 'google/apis/compute_v1'
+require 'google/cloud/compute/v1'
+require 'google-cloud-compute'
```

Comment:
I don't think it's related to Zeitwerk, those `require` statements can be used as they used to, since those are third-party libs, that's allowed :)

Although, if it helps to load them early in initializers and it works, I don't mind.

Reply:
I like the idea of having them all in one place. It was kind of surprising to see the `require` spread across the application.